packages/components/notification/src/NotificationItem/NotificationItemContainer.tsx

```diff
@@ -93,13 +95,13 @@ export class NotificationItemContainer extends Component<
           }
         }}
       >
-        <div
-          style={{ pointerEvents: 'all' }}
+        <Box
```

Comment:
I guess using `Box` here is an optional change. We don't pass any props outside, do it can stay as `div`